models/aether-2.0.x/yang/onf-application.yang

```diff
@@ -51,17 +51,13 @@ submodule onf-application {
       uses at:desc-display-name;
 
       leaf address {
-        type ietf:host;
+        type at:host-or-network;
         mandatory true;
         description
             "Address of this application. Either a hostname, an IP, or a subnet.";
       }
 
       list endpoint {
-        must "number(./ent:port-start) <= number(./ent:port-end)" {
```

Comment:
this erases my `must` rule - I guess I should have added it to the upstream `roc-helm-charts`